README.md

```diff
@@ -27,14 +28,16 @@ If the build fails because it can't find the `jni.h` header file, run it with:
 
 Changing the JDK directory to the one right for your system.
 
-*Note: currently only Linux is supported, check [#7](https://github.com/bblfsh/client-scala/issues/7) for MacOS support*.
+*Note: currently only Linux is supported, check
+[#7](https://github.com/bblfsh/client-scala/issues/7) for MacOS support*.
```

Comment:
As commented, the _hash 7_ already generates the link for you =)

Reply:
Ups, true.